webnn/conv2d.https.any.js

```diff
@@ -0,0 +1,49 @@
+// META: title=test WebNN API conv2d operation
+// META: global=window,dedicatedworker
+// META: script=./resources/utils.js
+// META: timeout=long
+
+'use strict';
+
+// https://webmachinelearning.github.io/webnn/#api-mlgraphbuilder-conv2d
+
+const buildConv2d= (operationName, builder, resources) => {
+  // MLOperand conv2d(MLOperand input, MLOperand filter, optional MLConv2dOptions options = {});
+  const namedOutputOperand = {};
+  const [inputOperand, filterOperand] = createMultiInputOperands(builder, resources);
+  const conv2dOptions = {};
+  const options = resources.options;
+  if (options) {
+    if (options.padding) {
+      conv2dOptions.padding = options.padding;
+    }
+    if (options.strides) {
+      conv2dOptions.strides = options.strides;
+    }
+    if (options.dilations) {
+      conv2dOptions.dilations = options.dilations;
+    }
+    if (options.autoPad) {
+      conv2dOptions.autoPad = options.autoPad;
+    }
+    if (options.groups) {
+      conv2dOptions.groups = options.groups;
+    }
+    if (options.inputLayout) {
+      conv2dOptions.inputLayout = options.inputLayout;
+    }
+    if (options.filterLayout) {
+      conv2dOptions.filterLayout = options.filterLayout;
+    }
```

Comment:
Too bad Javascript lacks a `??=` operator.

```
      conv2dOptions.padding ??= options.padding;
      conv2dOptions.strides ??= options.strides;
      conv2dOptions.dilations ??= options.dilations;
      conv2dOptions.autoPad ??= options.autoPad;
      conv2dOptions.groups ??= options.groups;
      conv2dOptions.inputLayout ??= options.inputLayout;
      conv2dOptions.filterLayout ??= options.filterLayout;
```

Oh well. (no action expected, just musing)

Reply:
unless I'm missing something, I think this could be accomplished with `conv2dOptions = {...options};`

Reply:
I've updated. Please take another look, thanks @dontcallmedom